examples/django_examples.py

```diff
@@ -146,43 +150,95 @@ def print_psql_instructions():
     )
 
 
-def query_example_match():
-    print(
-        '\nQuery example Customer.objects.get(name__match="ali"), which should find customer1 with "Alice Developer":'
-    )
-    record = Customer.objects.get(name__match="ali")
-
+def wait_and_show_record(record):
     print()
     input("Press Enter to continue.")
     print()
     print(f"  Record found: {record}")
     print()
 
 
+def query_exact_string():
+    print(
+        '\nQuery example Customer.objects.get(name__eq="Carol Customer"), which should find customer with the exact name "Carol Customer":'
+    )
+    record = Customer.objects.get(name__eq="Carol Customer")
+
+    wait_and_show_record(record)
+
+
+def query_example_match_string():
+    print(
+        '\nQuery example Customer.objects.get(name__match="ali"), which should find customer with the matching name "Alice Developer":'
+    )
+    record = Customer.objects.get(name__match="ali")
+
+    wait_and_show_record(record)
+
+
 def query_example_ore():
     print(
-        "\nQuery example Customer.objects.get(weight__gt=73.0), which should find customer2 with 82.1:"
+        "\nQuery example Customer.objects.get(weight__gt=73.0), which should find the customer with the weight 82.1:"
     )
     record = Customer.objects.get(weight__gt=73.0)
 
-    print()
-    input("Press Enter to continue.")
-    print()
-    print(f"  Record found: {record}")
-    print()
+    wait_and_show_record(record)
 
 
 def query_example_json_contains():
     print(
-        '\nQuery example Customer.objects.get(extra_info__contains={"key": []}), which should find customer1 with {"key": ["value"], "num": 1, "cat": "a"}:'
+        '\nQuery example Customer.objects.get(extra_info__contains={"key": []}), which should find the customer with the extra_info {"key": ["value"], "num": 1, "cat": "a"}:'
     )
     record = Customer.objects.get(extra_info__contains={"key": []})
 
-    print()
-    input("Press Enter to continue.")
-    print()
-    print(f"  Record found: {record}")
-    print()
+    wait_and_show_record(record)
+
+
+def query_example_exact_date():
+    print(
+        "\nQuery example Customer.objects.get(start_date_eq=date(2024, 1, 2)), which should find the customer with the start date 2024-01-02:"
+    )
+    record = Customer.objects.get(weight__gt=73.0)
```

Comment:
Looks like the query doesn't match the description here. Should this use the same query that gets `print`ed above?

Reply:
Thanks. Good point. I'll give see if/how I can do that.

Reply:
I ended up using `eval` which I think is acceptable for this example script

Reply:
Ah, I didn't necessarily mean that those two lines need to share code, but meant to point out that the description mentioned a query on start date but the actual query was for weight. Using `eval` or updating the description makes sense to me, but if it's a customer-facing example then we might want to avoid `eval` (to reduce the risk of `eval` examples getting copy/pasted into applications).

Reply:
Oh okay. I'd like to use `eval` here because I expect there will be more changes to this file and I will probably make similar mistakes if I manually copy-paste. While I agree we should not put `eval` in the examples, I also hope users of this library know they should not use it. Maybe I am a little naive there...